content/docs/news/release_announcements/keptn-0180/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Keptn 0.18.0
+weight: 69
+---
+
+# Release Notes [0.18.0](https://github.com/keptn/keptn/compare/0.17.0...0.18.0) (2022-07-28)
+
+---
+
+**Key announcements**:
+
+:information_source: *CLI*: The CLI no longer provides the install/uninstall/upgrade commands. Please refer to the [Install Keptn using the Helm chart](../../../install/helm-install/) to see how you can use Helm to install and operate Keptn.
+
+:mailbox: *WebHooks* support the new `v1beta1` version which provides a more structured way to create your webhooks. For futher information, see [Advanced Webhook configuration](../../../0.18.x/integrations/webhooks/#advanced-webhook-configuration).
+
+:star: *Bridge* support lazy loading to provide a faster response time.
+
+:sparkles: *(experimental) New Import API*: Keptn provides a new _alpha_ version of an import API. This API can be used to completely setup your Keptn project.
```

Comment:
From:

This API can be used to completely setup your Keptn project.

To:

 This API can be used to setup a Keptn project by importing a template as zip archive.